q2_feature_classifier/_skl.py

```diff
@@ -21,7 +21,8 @@
             'non_negative': True}],
           ['classify',
            {'__type__': 'custom.LowMemoryMultinomialNB',
-            'alpha': 0.01}]]]]
+            'alpha': 0.01,
+            'chunk_size': 20000}]]]]
```

Comment:
Specifically, I'm wondering if the error @gregcaporaso ran into (see the linked comment) is no longer raised when using the new default chunk size of 20000. He suggested choosing another chunk size (dynamically) in that case to avoid the error:

> basically we'll just need to detect when 20k isn't a sensible value, and go with something that is.